src/LavaNode.cs

```diff
@@ -300,10 +302,10 @@ await voiceChannel.DisconnectAsync()
     /// <returns></returns>
     public async Task<TLavaTrack> DecodeTrackAsync(string trackHash) {
         ArgumentNullException.ThrowIfNull(trackHash);
-        var responseMessage = await _httpClient.GetAsync($"/{_version}/decodetrack?encodedTrack={trackHash}");
+        var responseMessage = await _httpClient.GetAsync($"/{_version}/decodetrack?encodedTrack={HttpUtility.UrlEncode(trackHash)}");
         await using var stream = await responseMessage.Content.ReadAsStreamAsync();
         RestException.ThrowIfNot200(responseMessage.IsSuccessStatusCode, stream);
-        return await JsonSerializer.DeserializeAsync<TLavaTrack>(stream);
+        return await JsonSerializer.DeserializeAsync<TLavaTrack>(stream, new JsonSerializerOptions() { Converters = { new LavaTrackConverter() }});
```

Comment:
Better to create a static property for JsonSerializerOptions

Reply:
You're right, I had an issue while making it and forgot to change it back, fixed it now